lib/controllers/coinstac/consortia.js

```diff
@@ -4,10 +4,20 @@ const boom = require('boom');
 const joi = require('joi');
 const config = require('config'); // jshint ignore:line
 const _ = require('lodash');
+const Bluebird = require('bluebird');
 const consortiaDbOptions = config.get('coinstac.pouchdb.consortia');
+const cloudantClient = require('../../utils/cloudant-client.js');
+let isCloudant;
+
+if (config.has('coinstac.pouchdb.cloudant')) {
+    isCloudant = true;
+    cloudantClient.init();
```

Comment:
The cloudantClient is used only to set security rules on each remote cloudantDB after it is created. Without doing this, our Front End would need our username/pwd to connect to the remote DB.